src/traces/splom/style.js

```diff
@@ -0,0 +1,34 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+var calcColorscale = require('../scatter/colorscale_calc');
+var convertMarkerStyle = require('../scattergl/convert').markerStyle;
+
+function editStyle(gd, cd0) {
```

Comment:
Same here, I'm not this belong in a `style.js` file. I'd be ok we either:

- moving this logic to `splom/index.js`
- renaming this file to `edit_style.js`